Makefile

```diff
@@ -33,53 +35,54 @@ list:
 	@echo ${CITIES}
 
 %.osm.pbf:
+	@mkdir -p ${DATA_DIR}
 	@echo "Downloading $@ from BBBike.";
 	@echo "\n\nConsider donating to BBBike to help cover hosting! https://extract.bbbike.org/community.html\n\n"
 	wget -U headway/1.0 -O $@ "https://download.bbbike.org/osm/bbbike/$(notdir $(basename $(basename $@)))/$(notdir $@)" || rm $@
 
 %.bbox:
 	@echo "Extracting bounding box for $(notdir $(basename $@))"
-	grep "$(notdir $(basename $@)):" gtfs/bboxes.csv > $@
-	perl -i.bak -pe 's/$(notdir $(basename $@))://' $@
+	grep "$(notdir $(basename $@)):" gtfs/bboxes.csv > ${DATA_DIR}/bbox.txt
+	perl -i.bak -pe 's/$(notdir $(basename $@))://' ${DATA_DIR}/bbox.txt
 
 %.mbtiles: %.osm.pbf
-	@echo "Building MBTiles $(notdir $(basename $@))"
-	mkdir -p data/sources
-	cd data/sources && bash -c "find -name '*.zip' || (wget https://f000.backblazeb2.com/file/headway/sources.tar && tar xvf sources.tar)"
-	docker run --memory=$(DOCKER_MEMORY) --rm -e JAVA_TOOL_OPTIONS="$(JAVA_TOOL_OPTIONS)" \
-		-v "${DATA_DIR}:/data" \
-		ghcr.io/onthegomap/planetiler:latest \
-		--osm-path=/data/$(notdir $(basename $(basename $@))).osm.pbf \
-		--mbtiles=/data/$(notdir $(basename $(basename $@))).mbtiles \
-		--download \
-		--force
-
-%.nominatim.sql: %.osm.pbf
-	@echo "Bootstrapping geocoding index for $(basename $(basename $@))."
-	docker run --memory=$(DOCKER_MEMORY) -it --rm \
-		-v "${DATA_DIR}":/data \
-		-v "${DATA_DIR}/nominatim_pg/":/var/lib/postgresql/12/main \
-		-v "${DATA_DIR}/nominatim_flatnode/":/nominatim/flatnode \
-		-e PBF_PATH=/data/$(notdir $(basename $(basename $@))).osm.pbf \
-		mediagis/nominatim:4.0 \
-		bash -c 'useradd -m nominatim && /app/config.sh && /app/init.sh && touch /var/lib/postgresql/12/main/import-finished && service postgresql start && (sudo -u nominatim pg_dump nominatim > /data/$(notdir $(basename $(basename $@))).nominatim.sql)'
+	@echo "Building MBTiles $(basename $@)"
+	cp $(basename $@).osm.pbf mbtiles/data.osm.pbf
+	docker build ./mbtiles --tag headway_mbtiles_builder
```

Comment:
Should these images be tagged with the location name to avoid accidental mix-and-matching? like `--tag headway_mbtiles_builder_%`

Reply:
My worry with that is increased disk usage because then every image would be tagged. I can't really think of a way a mix-and-match situation would happen because we do a copy into the docker image's build context which should trigger a rebuild if one is necessary.

Reply:
Hm, how about changing the format to something like `headway_builder_mbtiles_%` and then we can add in `make clean_all` something like: `docker images ls | grep headway_builder_ | xargs docker rmi`.